ytypes/gnmi.go

```diff
@@ -35,7 +36,7 @@ import (
 //
 // If an error occurs during unmarshalling, schema.Root may already be
 // modified. A rollback is not performed.
-func UnmarshalNotifications(schema *Schema, ns []*gpb.Notification, opts ...UnmarshalOpt) error {
+func UnmarshalNotifications(schema *Schema, ns []*gpb.Notification, opts ...UnmarshalOpt) (*ComplianceErrors, error) {
```

Comment:
So this would be a backwards-incompatible change. I think it's better to make ComplianceErrors satisfy the error interface and return it instead. So essentially the flag controls whether this structure is returned.